render.rb

```diff
@@ -1,41 +1,29 @@
 #! /usr/bin/env ruby
 
 #
-# This script grabs the latest release details from the GitHub API for a project
-# and then generates a Homebrew Formula.
-#
-# It currently assumes you have separate binaries for Linux and macOS and generates
-# a forumula which works with both homebrew and linuxbrew.
-#
-# This is intended for cases where projects release quickly and you want a Homebrew Tap
-# to track those releases closely, and where you don't have automation (like
-# Goreleaser) which can generate Homebrew Formula automatically.
+# This script grabs the latest release details for a project
+# and then generates a scoop manifest.
 #
 
 require "erb"
 require "json"
 require "open-uri"
 
-PROJECT = "snyk/snyk"
 BIN = "snyk"
 
 # Return the URL, SHA and version information for a GitHub release
```

Comment:
suggestion: this comment is stale now (mentions GitHub) and not very useful (we can see return values in the return statement. Let's remove it?